README.md

```diff
@@ -1,5 +1,5 @@
 # .htaccess Snippets
-A collection of useful .htaccess snippets, all in one place. I decided to create this repo after getting so tired (and bored) with Googling everytime there's a need of forcing `www` for my new website.
```

Comment:
Seems valid to call them *snippets*, we should change all instance of `.htaccess snippets` to `.htaccess` if we really want to.